NomaiGrandPrix/SpawnPointMenu/SpawnPointSelectorManager.cs

```diff
@@ -219,6 +220,11 @@ public void OnItemClicked(PointerEventData eventData, SpawnPointMenuOption optio
             }
         }
 
+        public void Refresh()
+        {
+            Destroy(gameObject);
```

Comment:
What's causing this to be re-initialized?

Reply:
`_manager = SpawnPointSelectorManager.Instance;` in NomaiGrandPrix.cs ln 237

Reply:
Ah got it, thought this was NomaiGrandPrix before.